hucssley-classes.md

```diff
@@ -704,14 +703,14 @@ $hu-background-size-types: (
 );
 ```
 
-### hu-blend-mode
+### hu-mix-blend-mode
```

Comment:
Can you alphabetise this section please?

Reply:
Thanks for catching that.